signalilo/templates/deployment.yaml

```diff
@@ -27,10 +29,34 @@ spec:
       containers:
         - name: {{ .Chart.Name }}
           image: {{ .Values.image.registry }}/{{ .Values.image.repository }}:{{ .Values.image.tag }}
+          env:
+          - name: SIGNALILO_UUID
+            value: {{ required ".Values.config.uuid is required" .Values.config.uuid | quote }}
+          - name: SIGNALILO_ICINGA_HOSTNAME
+            value: {{ required ".Values.config.icinga_hostname is required" .Values.config.icinga_hostname | quote }}
+          - name: SIGNALILO_ICINGA_URL
+            value:  {{ required ".Values.config.icinga_url is required" .Values.config.icinga_url | quote }}
+          - name: SIGNALILO_ICINGA_USERNAME
+            value: {{ required ".Values.config.icinga_username is required" .Values.config.icinga_username | quote }}
+          - name: SIGNALILO_ICINGA_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: {{ .Values.config.icinga_password_secret | default (printf "%s" (include "signalilo.fullname" .)) | quote }}
+                key: "icinga_password"
+          - name: SIGNALILO_ALERTMANAGER_PORT
+            value: {{ .Values.config.alertmanager_port | quote }}
+          - name: SIGNALILO_ALERTMANAGER_BEARER_TOKEN
+            valueFrom:
+              secretKeyRef:
+                name: {{ .Values.config.alertmanager_bearer_token_secret | default (printf "%s" (include "signalilo.fullname" .)) | quote }}
+                key: "alertmanager_bearer_token"
+{{- if .Values.extraEnvVars }}
+{{ toYaml .Values.extraEnvVars | indent 10 }}
```

Comment:
I think you can leave out the `toYaml` and just indent the block `.Values.k8up_operator.extraEnvVars`.

Reply:
unfortunately that doesn't work as expected and produces `[map[name:SIGNALILO_DEBUG value:3]]` instead of YAML

Reply:
Oh, well. Never mind then.